### ISSUE TITLE
"Scenes" and "Settings" controls have no accessible name 

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3106]
The Menu button, "Scenes" and "Settings" controls are wrapped in a `<nav>` with class="_16EbidWL9cgqVGI4etuXZF", but it is not necessarily a true navigation  landmark, and no accessible name is provided for screen reader users to inform them  on the purpose of the region.

**Suggested Solution** 
A different grouping method may be more appropriate for the three controls instead of  a navigation landmark, such as a toolbar. Refer to the ARIA APG Toolbar Pattern:  https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/.  

Use an aria-label attribute on the widget container with a descriptive value to inform  screen reader users of how are the items in the group related.  

Severity level 2 from PAC

**Comment by WeirdRubberDuck** [comment]
Closed in #3608
